source/scripts/Scene.js

```diff
@@ -10,6 +11,7 @@ export default class Scene extends Pixi.Container {
         super()
 
         this.addChild(this.player = new Player())
+        this.addChild(this.foe = new Foe())
```

Comment:
Would you mind renaming it to `Baddie`, so it matches our other calls about baddies? Thoughts?